Lib/pathlib/_local.py

```diff
@@ -119,11 +119,23 @@ def __init__(self, *args):
         paths = []
         for arg in args:
             if isinstance(arg, PurePath):
-                if arg.parser is not self.parser:
+                if arg.parser is self.parser:
+                    paths.extend(arg._raw_paths)
+                elif arg.parser is ntpath:
                     # GH-103631: Convert separators for backwards compatibility.
+                    # GH-125012: This emits FutureWarning as of Python 3.14.
+                    import warnings
+                    msg = ("pathlib.PurePosixPath(pathlib.PureWindowsPath(...)): "
```

Comment:
Excuse my OCD, can you align them?
```
("xx"
 "xx"
 "xx")
```

Reply:
Can you make a suggestion? (ctrl-g). I don't follow what you're asking, sorry.